### ISSUE TITLE
Encoding Loss Error

### ISSUE THREAD

**Issue by G-1nOnly** [RaduAlexandru/permutohedral_encoding#6]
Hi, thanks for the wonderful work! I'm currently using permutohedral encoding in my SDF training pipeline, but the loss appears to be NaN, is there any possible reasons? I'm using only 1 RTX3090 for training and the learning rate is 1e-3 in my model.
